omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/dashboardwidgets/DashboardWidgetFragmentController.java

```diff
@@ -50,11 +50,16 @@ public void controller(FragmentConfiguration config, @FragmentParam("app") AppDe
         if (appConfig.get("JSDateFormat") == null) {
             appConfig.put("JSDateFormat", adminService.getGlobalProperty(UiFrameworkConstants.GP_FORMATTER_JS_DATE_FORMAT, "YYYY-MMM-DD"));
         }
+
+        if (appConfig.get("JSDateTimeFormat") == null) {
```

Comment:
What happens when the global property is changed at runtime? Since the JSDateTimeFormat in appConfig will not be null, how will once change this value without restarting OpenMRS?

Reply:
That is a good question, on this case I follow the behavior of `JSDateFormat` and `dateFormat `above. It is suppose to be possible to change that GP on runtime?

Reply:
Just go to the settings page and see how you can change a global property at runtime.

Reply:
I just made some tests it works fine.
So that appConfig.get will check if in the widget configuration we specify some format if we dont specify any, it will try to get the format from the GP, if the global property is null or not present It has a fallback value "YYYY-MMM-DD HH:mm".
So even if we remove the GP it has the fallback value.

Reply:
But if we change the global property, at runtime, to a different value, will this value be used without having to restart the application?

Reply:
Yes it will. If we don't specify the format on the widget config.

Reply:
Did you test and confirm it?

Reply:
Yes, i test it and confirm it.